source_pubsub/source_test.go

```diff
@@ -45,7 +45,7 @@ func Test_Read(t *testing.T) {
 		ClientID:      "test-client-id",
 		ClientSecret:  "test-client-secret",
 		OAuthEndpoint: "https://somewhere",
-		TopicName:     "/events/TestEvent__e",
+		TopicNames:    []string{"/events/TestEvent__e"},
```

Comment:
nice. would be great to also add a test case here with 2 topics - thought, you might already be on this since it's in draft mode